Automattic/woocommerce-payments
# Downgrade PHP polyfill packages to support PHP 7.0.

Ref: p1643103104242100-slack-C01BPL3ALGP
Fixes this error for PHP7.0. 

> [25-Jan-2022 08:24:32 UTC] PHP Parse error:  syntax error, unexpected '?', expecting variable (T_VARIABLE) in ./public/wp-content/plugins/woocommerce-payments/vendor/symfony/polyfill-php80/bootstrap.php on line 29
>
> [25-Jan-2022 08:24:32 UTC] PHP Fatal error:  Exception thrown without a stack frame in Unknown on line 0


#### Changes proposed in this Pull Request

- Downgrade polyfill packages to `1.19.0` as since `1.20.0` they require PHP 7.1+. E.g. https://packagist.org/packages/symfony/polyfill-php80#v1.20.0
- The package for PHP 8.1 only supports PHP7.1+. https://packagist.org/packages/symfony/polyfill-php81#v1.22.0









#### Testing instructions





* Since we're not having CI tests for PHP 7.0 as mentioned in https://github.com/Automattic/woocommerce-payments/issues/2970#issuecomment-1015598461, we may need to check this manually. For example, create a Jurassic site with PHP 7.0 https://jurassic.ninja/specialops/#
* 

-------------------


**Post merge**
